README.md

```diff
@@ -13,31 +13,35 @@ A modular build system for hardware ("make for silicon"). The project philosophy
 
 # Why SiliconCompiler?
 
-* **Ease-of-use**: Programmable with a simple [Python API](https://docs.siliconcompiler.com/en/latest/user_guide/programming_model.html)
-* **Portability:** Powerful dynamic JSON [schema](https://docs.siliconcompiler.com/en/latest/reference_manual/schema.html) supports ASIC and FPGA design and simulation
-* **Speed:** Flowgraph [execution model](https://docs.siliconcompiler.com/en/latest/user_guide/execution_model.html) enables cloud scale execution.
-* **Friction-less:** [Remote execution model](https://docs.siliconcompiler.com/en/latest/user_guide/remote_processing.html) enables "zero install" compilation
-* **Modularity:** [Tool abstraction layer](architecture) makes it easy to add/port new tools to the project.
-* **Provenance:** [Comilation manifests](https://docs.siliconcompiler.com/en/latest/user_guide/data_model.html) created automatically during execution.
-* **Documented:** An extensive set of auto-generated high quality [reference documents](https://docs.siliconcompiler.com/en/latest/).
+* **Ease-of-use**: Programmable with a simple [Python API](https://docs.siliconcompiler.com/en/stable/user_guide/programming_model.html)
+* **Portability:** Powerful dynamic JSON [schema](https://docs.siliconcompiler.com/en/stable/reference_manual/schema.html) supports ASIC and FPGA design and simulation
+* **Speed:** Flowgraph [execution model](https://docs.siliconcompiler.com/en/stable/user_guide/execution_model.html) enables cloud scale execution.
+* **Friction-less:** [Remote execution model](https://docs.siliconcompiler.com/en/stable/user_guide/remote_processing.html) enables "zero install" compilation
+* **Modularity:** [Tool abstraction layer](https://docs.siliconcompiler.com/en/stable/user_guide/tools.html) makes it easy to add/port new tools to the project.
+* **Provenance:** [Compilation manifests](https://docs.siliconcompiler.com/en/stable/user_guide/data_model.html) created automatically during execution.
+* **Documented:** An extensive set of auto-generated high quality [reference documents](https://docs.siliconcompiler.com/).
 * **In-use:** Actively used by Zero ASIC for commercial tapeouts at advanced process nodes.
 
 # Supported Technologies
 
 | Type | Supported|
 |------|----------|
-|**Languages**| C, SV, VHDL, Chisel, Migen/Amaranth, Bluespec
+|**Languages**| C, Verilog, SV, VHDL, Chisel, Migen/Amaranth, Bluespec
 |**Simulation**| Verilator, Icarus, GHDL
 |**Synthesis**| Yosys, Vivado, Synopsys, Cadence
 |**ASIC APR**| OpenRoad, Synopsys, Cadence
 |**FPGA APR**| VPR, nextpnr, Vivado
-|**Layout Viewer**| Klayout, Cadence, Synopsys
+|**Layout Viewer**| Klayout, OpenRoad, Cadence, Synopsys
 |**DRC/LVS**| Magic, Synopsys, Siemens
 |**PDKs**| sky130, asap7, freepdk45, gf12lp, intel16
 
```

Comment:
In the getting started section, I think it should be `python -m pip install --upgrade siliconcompiler`
instead of `python -m pip upgrade siliconcompiler`

Reply:
@gadfort this is not a change of yours but I noticed it when I was just reviewing the README in general, with your change in them.